packages/bap-cxxfilt/bap-cxxfilt.master/opam

```diff
@@ -24,7 +23,7 @@ depends: [
   "ocaml" {>= "4.07.0" & < "4.10.0"}
   "bap-std" {= "master"}
   "bap-demangle"
-  "conf-binutils" {>= "0.2"}
+  "conf-binutils" {= "master"}
```

Comment:
What is the motivation to use "master" here instead of an explicit "0.3"?

Reply:
no motivation, just thought it would be better